src/classes/STG_InstallScript.cls

```diff
@@ -52,6 +52,9 @@ global without sharing class STG_InstallScript implements InstallHandler {
             List<TDTM_Global_API.TdtmToken> defaultTokens = TDTM_Global_API.getDefaultTdtmConfig();
             TDTM_Global_API.setTdtmConfig(defaultTokens, 'hed');
 
+            // Enable new Preferred Phone sync functionality
+            enablePreferredPhoneSync();
```

Comment:
Since the default value of Enable New Preferred Phone Sync is false, we set it to true here when installing EDA for a new customer

Reply:
Instead of running enablePreferredPhoneSync() for initial installation, can we use configSettings() to set the value of this custom setting to "true"?

Reply:
I'm not sure here. I like that the unit tests don't have it enabled by default as this helps us deal with the other issue we had last release with customer unit tests breaking.

Thoughts @lifewithryan @ceiroa ?

Reply:
I hear you @xinnanlisf, but I also like the tests not having it on by default. I think it's ok to have the class be long if it's not getting more complex. Calling a bunch of methods in order is not a big deal if we aren't increasing cyclomatic complexity. We can also refactor and rearrange/cleanup at any time.